dev-libs/caf/caf-0.17.3.ebuild

```diff
@@ -0,0 +1,116 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_ECLASS=cmake
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit cmake-multilib python-single-r1
+
+DESCRIPTION="The C++ Actor Framework (CAF)"
+HOMEPAGE="https://actor-framework.org/"
+SRC_URI="https://github.com/actor-framework/actor-framework/archive/${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="|| ( Boost-1.0 BSD )"
+SLOT="0/17.3"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug doc examples opencl +openssl python static-libs test tools"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="examples? ( net-misc/curl[${MULTILIB_USEDEP}] )
+	openssl? ( dev-libs/openssl:0=[${MULTILIB_USEDEP},static-libs?] )
+	opencl? ( virtual/opencl[${MULTILIB_USEDEP}] )
+	python? ( ${PYTHON_DEPS}
+		dev-python/ipython
+		dev-python/pybind11[${PYTHON_USEDEP}] )"
+
+DEPEND="${RDEPEND}"
+
+BDEPEND="doc? ( app-doc/doxygen[dot]
+	app-shells/bash:0
+	dev-texlive/texlive-fontsextra
+	dev-texlive/texlive-latex
+	dev-texlive/texlive-latexextra )"
+
+RESTRICT="!test? ( test )"
+
+src_unpack() {
+	default
+	mv * "${P}" || die
```

Comment:
Why? Wouldn't it be enough to override $S?

Reply:
Done